.gitignore

```diff
@@ -2,6 +2,7 @@
 .godot/
 .DS_Store
 /.build
+/.index-build
```

Comment:
Swift's VSCode plugin has a background indexing option, which creates this folder.